src/scripts/experience/issue.js

```diff
@@ -20,10 +22,16 @@ define([
         this._canvasSize = canvasSize;
         this.elm = new PIXI.DisplayObjectContainer();
 
+        // Track in global list of issues.
+        this.id = _.uniqueId();
+        Issue._allIssues[this.id] = this;
+
         this.mouseOverS = new signals.Signal();
         this.mouseOutS = new signals.Signal();
         this.pressS = new signals.Signal();
         this.tapS = new signals.Signal();
+        this.focusS = new signals.Signal();
```

Comment:
Issues have a set of signals on them that the rest of the app triggers at appropriate times (mostly it's `experience-canvas.js` that is doing the triggering, as that file seems to be the glue between the DOM and the higher-level logic). It seemed natural to just add focus and blur to the list of things that can be triggered and subscribed to.